apps/frontend/components/pages/Form/FormValidationContainer.tsx

```diff
@@ -28,7 +28,7 @@ export const FormValidationContainer = (): JSX.Element => {
         <FormValidation validationForm={form} />
       )}
       <Box display="flex" flexDirection="column" alignItems="center">
-        <ValidationIndicator valStatus={form?._validation_status.uid} />
```

Comment:
let's do the check here Validation Indicator should not be aware of the full form